runelite-client/src/main/java/net/runelite/client/plugins/mta/MTAConfig.java

```diff
@@ -81,18 +88,57 @@ default boolean alchemy()
 	@ConfigItem(
 			keyName = "enchantmentEnable",
 			name = "Enable",
-			description = "Configures whether or not the enchantment room overlay is enabled.",
+			description = "Configures whether or not the Enchanting Chamber overlay is enabled.",
+			position = 0,
 			section = enchantmentGroup
 	)
 	default boolean enchantment()
 	{
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "highlightDragonstones",
+			name = "Highlight dragonstones",
+			description = "Enable markers indicating spawned dragonstones.",
+			position = 1,
+			section = enchantmentGroup
+	)
+	default boolean highlightDragonstones()
```

Comment:
Add a "No marking" marking mode and this config is no longer needed.

Reply:
Good point, implemented in MTAConfig.